connPFM/tests/test_ev.py

```diff
@@ -28,34 +28,110 @@ def test_rss_surr(AUC_file, atlas_file, surr_dir, rssr_auc_file):
     )
 
     AUC_img = masker.fit_transform(AUC_file)
-    _, u, v = ev.calculate_ets(AUC_img, AUC_img.shape[1])
-    rssr, _, _ = ev.rss_surr(AUC_img, u, v, join(surr_dir, "surrogate_AUC_"), "", masker, 0)
+    _, u, v = connectivity_utils.calculate_ets(AUC_img, AUC_img.shape[1])
+    rssr, _, _ = connectivity_utils.rss_surr(
+        AUC_img, u, v, join(surr_dir, "surrogate_AUC_"), "", masker, 0
+    )
     rssr_auc = np.loadtxt(rssr_auc_file)
     assert np.all(np.isclose(rssr, rssr_auc))
 
 
 def test_threshold_ets_matrix():
+    thr = 2
     dum_mat = np.ones((3, 3))
     dum_mat[2, 1] = 3
     dum_mat2 = np.zeros((3, 3))
-    dum_mat2[2, 1] = 3
-    th_dum = ev.threshold_ets_matrix(dum_mat, [2], 2)
-    assert np.all(th_dum == dum_mat2)
+    dum_mat2[2, 1] = 3 - thr
```

Comment:
shouldn't this be dum_mat2
```suggestion
    dum_mat2[2, 1] = 1
```